docs/get-started/add-shaper.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Add a shaper'
+sidebar_position: 5
+---
+
+# Add shaper
+
+**Shapers** are nodes that are usually uniquely connected as head nodes, meaning that they receive 
+data inputs but will output no data to another node. Developers can define the structure of output 
+data when creating the source node.
+
+We will connect it to our **transformation** in the future.
+
+To create a new **shaper** to our project we can use the sub-command `holium shaper create`:
+
+```shell
+$ holium shaper create sales_results --json-schema '{\"type\":\"array\", \"prefixItems\":[{\"type\":\"array\", \"items\": {\"type\":\"array\", \"prefixItems\": [{\"type\": \"string\"}, {\"type\": \"number\"}]}}]}'
+new object created: sales_results
+```
+
+Once it is done we are able to list all of our **shapers** with the sub-command `holium shaper list`:
+
+```shell
+$ holium shaper list
+┌───────────────┬──────────────────────────────┐                  
```

Comment:
Seems like a wrong output for the given example.

Reply:
Copied back all print command result from own CLI after a working running test in https://github.com/polyphene/docs.holium.org/pull/16/commits/054409ef8ae5bf0ac357484526e9e5038f708596